File: Back/src/ProjetoAngular.Domain/Evento.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProjetoAngular.Domain
{
    public class Evento
    {
        public int Id { get; set; }
        public string Local { get; set; }
        public DateTime? DataEvento { get; set; }
        public string Tema { get; set; }
        public int QuantidadePessoas { get; set; }
        public string ImagemURL { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public IEnumerable<Lote> Lote { get; set; }
        public IEnumerable<RedeSocial> RedeSocial { get; set; }
        public IEnumerable<PalestranteEvento> PalestranteEvento { get; set; }


    }
}

File: Front/ProjetoAngular-App/src/app/models/Evento.ts
import { Lote } from "./Lote";
import { Palestrante } from "./Palestrante";
import { RedeSocial } from "./RedeSocial";

export interface Evento
{

     id: number;
     local: string ;
     dataEvento?: Date;
     tema: string ;
     quantidadePessoas: number;
     imagemURL: string ;
     telefone: string ;
     email: string ;
     lote: Lote[];
     redeSocial: RedeSocial[] ;
     palestranteEvento: Palestrante[] ;

}
